specification/summary-care-record.yaml

```diff
@@ -154,14 +154,66 @@ paths:
           content:
             application/fhir+json:
               schema:
-                $ref: '#/components/schemas/OperationOutcome'
+                $ref: components/schemas/OperationOutcome.yaml
               example:
                 resourceType: OperationOutcome
                 issue:
                   - severity: error
                     code: invalid
                     details:
                       text: Internal error
+                      
+  /DocumentReference:
+    get:
+      summary: Get patient's latest Summary Care Record identifier
+      operationId: get-scr-id
+      tags:
+        - SCR
+      parameters:
+        - $ref: '#/components/parameters/BearerAuthorization'
+        - $ref: '#/components/parameters/CorrelationID'
+        - $ref: '#/components/parameters/RequestID'
+        - in: query
+          name: patient
+          description: The patient's NHS Number. Must be preceded with FHIR identifier (eg."patient=https://fhir.nhs.uk/Id/nhs-number|9995000180")
+          required: true
+          example: https://fhir.nhs.uk/Id/nhs-number|9995000180
+          schema:
+            type: string
+            
+        - in: query
+          name: type
+          description: General Practice Summary snomed type. Must be equal "type=http://snomed.info/sct|196981000000101"
+          required: false
+          example: http://snomed.info/sct|196981000000101
+          schema:
+            type: string
+          
+        - in: query
+          name: _sort
+          description: Defines how Patient's SCR list should be sorted in order to retrieve the latest one. The only supported value is _sort=date
+          required: false
+          example: date
+          schema:
+            type: string
+             
+        - in: query
+          name: _count
+          description: Defines the number of latest patient SCR IDs that sgould be retrieved. Currently the only supported value is _count=1.
```

Comment:
Will this throw 400 if other value is passed in?

Reply:
Yes. I added some additional description so it's clear.